src/MOIWrapper.jl

```diff
@@ -71,266 +52,204 @@ const solveoptionmap = Dict(
    :InfeasibleReturn => set_infeasible_return,
    )
 
-function setoption(m::ClpOptimizer, name::Symbol, value)
+function setoption(optimizer::ClpOptimizer, name::Symbol, value)
     if haskey(optionmap, name)
-        optionmap[name](m.inner,value)
+        optionmap[name](optimizer.inner,value)
     elseif haskey(solveoptionmap, name)
-        solveoptionmap[name](m.solveroptions,value)
+        solveoptionmap[name](optimizer.solveroptions,value)
     else
         error("Unrecognized option: $name")
     end
 end
 
 function ClpOptimizer(;kwargs...)
-    m = ClpOptimizer(nothing)
-    m.params = Dict{String,Any}()
-    MOI.empty!(m)
+    optimizer = ClpOptimizer(nothing)
+    optimizer.params = Dict{String,Any}()
+    MOI.empty!(optimizer)
     for (name,value) in kwargs
-        m.params[Symbol(name)] = value
+        optimizer.params[Symbol(name)] = value
     end
-    return m
+    return optimizer
 end
 
-LQOI.LinearQuadraticModel(::Type{ClpOptimizer},env) = ClpModel() 
+LQOI.LinearQuadraticModel(::Type{ClpOptimizer},env) = ClpModel()
 
-LQOI.supported_constraints(s::ClpOptimizer) = SUPPORTED_CONSTRAINTS
-LQOI.supported_objectives(s::ClpOptimizer) = SUPPORTED_OBJECTIVES
+LQOI.supported_constraints(optimizer::ClpOptimizer) = SUPPORTED_CONSTRAINTS
+LQOI.supported_objectives(optimizer::ClpOptimizer) = SUPPORTED_OBJECTIVES
 
 """
-    change_variable_bounds!(instance, cols::Vector{Int}, 
-        values::Vector{Float64}, senses::Vector)
-        
-Change the bounds of the variable.
+    replace_inf(x::Vector{<:Real})
+
+Replaces any occurances of an element `x[i]>1e20` with `Inf` and `x[i]<-1e20`
+with `-Inf`.
+"""
+function replace_inf(x::Vector{<:Real})
+    for i in 1:length(x)
+        if x[i] > 1e20
+            x[i] = Inf
+        elseif x[i] < -1e20
+            x[i] = -Inf
+        end
+    end
+    return x
+end
+
 """
-function LQOI.change_variable_bounds!(instance::ClpOptimizer, cols::Vector{Int}, 
+    replace_inf(x::Real)
+
+Return `Inf` if `x>1e20`, `-Inf` if `x<-1e20`, and `x` otherwise.
+"""
+function replace_inf(x::Real)
+    if x > 1e20
+        return Inf
+    elseif x < -1e20
+        return -Inf
+    else
+        return x
+    end
+end
+
+function LQOI.change_variable_bounds!(instance::ClpOptimizer, cols::Vector{Int},
         values::Vector{Float64}, senses::Vector)
-    
-    upperbounds = replace_inf(get_col_upper(instance.inner))    
-    lowerbounds = replace_inf(get_col_lower(instance.inner))    
-    
+    upperbounds = replace_inf(get_col_upper(instance.inner))
+    lowerbounds = replace_inf(get_col_lower(instance.inner))
     for i in 1:length(senses)
         if senses[i] == Cchar('U')
             upperbounds[cols[i]] = values[i]
         elseif senses[i] == Cchar('L')
             lowerbounds[cols[i]] = values[i]
-        else            
-            error("sense is " * string(senses[i]) * ", but only " * Cchar('U') *  
-            " and " * Cchar('L') * " senses are supported")
```

Comment:
I have an issue with this since the Cchar('L') is not 'L'. The string should either use the value of Cchar('L') or the string "Cchar('L')"